functions.py

```diff
@@ -170,11 +170,35 @@ def get_jenkins_job_info(server, job_name):
 	return jenkins_api_info
 
 
-def get_jenkins_jobs(server, job_search_fields):
+def get_jenkins_jobs(server, job_search_fields, job_search_fields_regex=""):
 	''' takes in a Jenkins server object and job_search_fields string
 		returns list of jobs with given search field as part of their name
 	'''
 
+	# does job_search_fields_regex contain valid regex?
+	is_valid_regex = False
+
+	if (job_search_fields_regex is not None):
```

Comment:
```suggestion
	if (job_search_fields_regex):
```

Reply:
Also remove parenthesis

Reply:
Also, make it consistent across all your change  (I had thought these parentheses were consistent across Jeeves)